test/unit/app/clone.spec.ts

```
@@ -10,8 +10,7 @@ import {
     defineCoreHandler,
 } from '../../../src';
 import type { IRouter, Route, RouteMatch } from '../../../src';
```

ESLint

[error] '../../../src' imported multiple times.

(import/no-duplicates)